dpdk/vr_dpdk_interface.c

```diff
@@ -392,8 +483,6 @@ dpdk_if_add(struct vr_interface *vif)
     } else if (vif_is_vhost(vif)) {
         return dpdk_vhost_if_add(vif);
     } else if (vif->vif_type == VIF_TYPE_AGENT) {
-        /* TODO: remove xconnect */
-        vhost_remove_xconnect();
```

Comment:
We call vhost_remove_xconnect() in dp-core vrouter_setup_vif(). I used the code above to debug the vRouter, so we do not need it anymore.